chain/epoch-manager/src/validator_selection.rs

```diff
@@ -449,7 +448,7 @@ mod tests {
             Default::default(),
             Balance::ZERO,
             PROTOCOL_VERSION,
-            epoch_config.shard_layout.clone(),
+            ShardLayout::multi_shard(2, 0),
```

Comment:
hardcoding the shard layout seems like a bad idea, can you read it from the epoch info?